app/views/posts/partials/show/_avoid_posting.html.erb

```diff
@@ -0,0 +1,33 @@
+<% if post.avoid_posting_artists.any? %>
+  <div class="notice" id="avoid-posting-notice">
+    <div class="avoid-posting">
+      <h3>Avoid Posting</h3>
+      <ul>
+        <% post.avoid_posting_artists.each do |dnp| %>
+          <% verified = dnp.artist && dnp.artist.linked_user_id == post.uploader_id %>
+          <li>
+          <% if dnp.details.blank? %>
+            <% if verified %>
```

Comment:
It seems to me that it should display "Only the artist is allowed to post" whenever:
- The DNP details are blank
- An account is linked to the artist page

In your implementation, the DNP reason would change between "Only the artist is allowed to post" and "Avoid posting" depending on whether the artist account ID matches that post's uploader's ID. While technically accurate, I feel like this would create confusion for the janitors.